tests/features/capsulahub-application.feature

```diff
@@ -0,0 +1,182 @@
+Feature: Build and Run commands for CapsulaHub CLI
+
+  Background:
+    Given an npm package called @capsulajs/capsula-hub
+    And   a default port 55555
+    And   a default configProvider httpFile
+    And   the following <configProvider> available
+          |<configProvider>                   |
+          |LocalFileConfigurationProvider     |
+          |HttpFileConfigurationProvider      |
+          |HardcoreServerConfigurationProvider|
+          |HttpServerConfigurationProvider    |
+          |LocalStorageConfigurationProvider  |
+    And   a default path to the output folder "./dist"
+    And   a valid port is a number between 1024 and 65535
+
+  Scenario: Run Capsula-Hub instance with specifying valid arguments
+    Given a valid token with configuration
+    When  I run the command `capsula-hub run --token="token" --configProvider="configProvider" --dispatcherUrl="dispatcherUrl" --port="port"` #add dispatcherUrl only for hardcoreServer provider
+    Then  an workspace is created
```

Comment:
Not sure that we are able to manually check hardcoreServer now. Or is it possible, @stephanebenayoun ?

Reply:
Anyway our tests are running with mocks no ?

Reply:
I see, we will run with mocks

Reply:
I think we need this test for each configurationProvider - and the token will be unique for each of providers